### ISSUE TITLE
highway=path nicht indiziert ?

### ISSUE THREAD

**Issue by jotpe** [abrensch/brouter#45]
I discovered a short [path](http://www.openstreetmap.org/way/48047868) which seems not be routeable. Look this [query](http://brouter.de/brouter-web/#zoom=19&lat=50.906547&lon=7.014947&layer=OpenStreetMap&lonlats=7.014998,50.906326|7.015108,50.906373&nogos=&alternativeidx=0&format=geojson&profile=hiking-beta).

Is something wrong?

**Comment by abrensch** [comment]
yes and no. What's not routable is a 4m section of railway=platform.

Probably the bike- and foot-profiles should allow routing over railway=platform?

You can add that at the very beginning of the costfactor-calculation, e.g.:

  if railway=platform then 8 else

I leave this issue open as a reminder to change the default profiles

**Comment by jotpe** [comment]
> Probably the bike- and foot-profiles should allow routing over railway=platform?

Yes, please.

**Comment by abrensch** [comment]
>> Probably the bike- and foot-profiles should allow routing over railway=platform?
> Yes, please.

it's not straight forward, because just making railway=platform walkable would create too many "islands".

That would require a concept to exclude such islands in wayoint matching
